pyvista/core/dataset.py

```diff
@@ -2206,6 +2271,7 @@ def find_containing_cell(self, point: Union[Sequence, np.ndarray]) -> Union[int,
         -1
 
         Find the cells that contain 1000 random points inside the mesh.
+
```

Comment:
This was the only thing broken in the build, and this wasn't even done in this PR originally. If CI is green this is good to merge, thanks.